core/src/main/java/io/ultrabrew/metrics/Histogram.java

```diff
@@ -0,0 +1,17 @@
+package io.ultrabrew.metrics;
+
+import io.ultrabrew.metrics.util.DistributionBucket;
+
+public class Histogram extends Timer {
```

Comment:
Histogram should extend Gauge, not Timer. While most use cases of Histogram will be related to Timer/Latencies, there do exist use cases for non-time based measurements like object sizes, message size, bytes transferred, time elapsed (non-Timer use case) since connections were established, etc.

Reply:
Monitoring tools such as _statsd_ use "histogram" and "timer" as synonyms. Confusion is inevitable. Whatever the histogram-enabled class is called, it is-a gauge and therefore should extend it.

Reply:
That's a long-winded way of saying I agree but not necessarily for the same reason. Having `Timer` distinct from `Histogram` doesn't make sense to me yet. You want some other type indicating that one or more data sketches (of which a histogram is only one example) are to be applied to measurements of the metric identity.